packages/utils/perf/score-report/optimized0.ts

```diff
@@ -45,7 +45,7 @@ function categoryRefToScore(
         }
         return group.score;
       default:
-        throw new Error(`Type ${ref.type} is unknown`);
+        throw new Error(`Type ${String(ref.type)} is unknown`);
```

Comment:
What is the type of `ref.type` here. If it's `never`, we should remove the `default` case. It's unreachable, and will prevent TypeScript warning us if we forget to handle some possible value here.

(Same goes for other `optimized*.ts` files.)

Reply:
> (Same goes for other optimized*.ts files.)
Not sure I understand. This logic is only in `optimized0`.

Reply:
> > (Same goes for other optimized*.ts files.)
>
> Not sure I understand. This logic is only in optimized0.


My mistake, I put this in the wrong comment :man_facepalming: Was meant to be here I think: https://github.com/code-pushup/cli/pull/422#discussion_r1456001506